org.eclipse.dawnsci.analysis.dataset/src/org/eclipse/dawnsci/analysis/dataset/slicer/SliceInformation.java

```diff
@@ -100,6 +100,25 @@ public void reducedDimensionToSingular(int dim) {
 		sampling = new SliceND(sss,ss);
 	}
 	
+	public void convertSliceDimensionToFull(int dim) {
+		
+		Slice[] cs = output.convertToSlice();
+		int[] sss = sampling.getShape();
+		Slice[] ss = sampling.convertToSlice();
+		
+//		sss[dim] = 1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)